src/page/home/community/community.tsx

```diff
@@ -0,0 +1,118 @@
+import type { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Card } from '@/component/card/card'
+import { useMemo } from 'react'
+import { shallow } from 'zustand/shallow'
+import { Icon } from '@/component/icon/icon'
+import { useStatsAggregate } from '@/domain/stats/aggregate'
+import { useAppStore } from '@/store/appStore'
+import '../i18n/index'
+import './community.scss'
+
+type CommunityProps = {
+  className?: string
+}
+
+type StatProps = {
+  description: string
+  stat: number
+  icon?: JSX.Element
+}
+
+type SingleStat = StatProps & {
+  _tag: 'singlestat'
+}
+
+type MultiStat = {
+  _tag: 'multistat'
+  stats: StatProps[]
+}
+
+type StatCard = SingleStat | MultiStat
+
+const Stat: FC<StatProps> = ({ description, stat, icon }): JSX.Element => (
+  <div className="okp4-dataverse-portal-home-page-stat-main">
+    {icon && <div className="okp4-dataverse-portal-home-page-stat-icon-container">{icon}</div>}
+    <div className="okp4-dataverse-portal-home-page-stat-content-wrapper">
+      <h2>{stat}</h2>
+      <p>{description}</p>
+    </div>
+  </div>
+)
+
+// eslint-disable-next-line max-lines-per-function
+export const Community: FC<CommunityProps> = ({ className }) => {
```

Comment:
Good point. In order to position the cards in the grid layout of the block container `Community` is now returning a fragment. I've also had to wrap the multistat card with the `with-theme` mixin because otherwise the width from the card component takes priority.

Reply:
Will be fixed in another PR.